config/initializers/content_store.rb

```diff
@@ -0,0 +1,14 @@
+require 'gds_api/content_store'
+
+class GdsApi::ContentStore::Fake
+  def content_item!(tag)
+    Struct.new("ContentItem", :content_id)
+    Struct::ContentItem.new("#{tag}-content-id")
+  end
+end
+
+Whitehall.content_store = if Rails.env.test?
+                            GdsApi::ContentStore::Fake.new
+                          else
+                            GdsApi::ContentStore.new(Plek.find('content-store'))
```

Comment:
I was having network error problems on CI without this… So I tried matching what `contentapi` does, which worked. As far as I can see on CI, some tests use the real content api, `@real_content_api` tag. I hope this is the correct thing to do, too.